src/api/user/server.js

```diff
@@ -0,0 +1,12 @@
+const { logger } = require('@senecacdot/satellite');
+const path = require('path');
+const service = require('.');
+require('dotenv').config({ path: path.join(__dirname, '../env.development') });
```

Comment:
This line is probably not needed. The developer should have a .env file made using env.development during dev